index.js

```diff
@@ -85,6 +85,10 @@ var botFunctions = {
     condition: /\/hug/,
     response: botResponseHug
   }
+  dunno: {
+    condition: /\I dunno|I don't know|I dont know/,
```

Comment:
I reckoned that without the `I` this will response would come up pretty often when its not related to the comment.